bin/commands/migrate/for/kubernetes/index.sh

```diff
@@ -56,7 +56,7 @@ original_zss_port="${ZWE_components_zss_port}"
 # prepare target zowe.yaml for k8s
 cp "${ZWE_CLI_PARAMETER_CONFIG}" "${temp_dir}/zowe.yaml"
 
-if [ "${ZWE_zowe_certificate_keystore_type}" = "JCERACFKS" ]; then
+if [[ "${ZWE_zowe_certificate_keystore_type}" == JCE*KS ]]; then
```

Comment:
I guess it is the same as `index.sh`, Let's say it is a question if validation of schema is enough or not. If not we should define all known store types there. If we count on schema it will be easier to maintain - just one place to add a new type.